src/guigame.c

```diff
@@ -1134,6 +1145,10 @@ void guiGameRemoveSettings(config_set_t *configSet)
         configRemoveKey(configSet, CONFIG_ITEM_ENABLECHEAT);
         configRemoveKey(configSet, CONFIG_ITEM_CHEATMODE);
 
+        //OSD Language
```

Comment:
Need to remove the source cfg key here also, pad macro should also remove source if you wanted to add that in too.